src/Controller/AboutController.php

```diff
@@ -120,27 +120,28 @@ public function peerReviewAction(Request $request) : Response
-                'the public reviews, revision recommendations, and author responses are published with accepted manuscripts.',
+                'we only review research papers that have been made available as preprints',
+                'all decisions are made by editors who are active researchers',
+                'we do not artificially limit the number of articles we publish or have a set acceptance rate',
+                'editors and reviewers discuss their reviews with each other before reaching a decision on a manuscript; extra experiments, analyses, or data collection are only requested if they are essential and can be reasonably completed within about two months (please also see, “<a href="'.$this->get('router')->generate('article', ['id' => '57162']).'">Publishing in the time of COVID-19</a>”)',
+                'our editorial process produces two outputs: i) public comments from the peer reviewers on the strengths and weaknesses of the work; ii) detailed feedback on the manuscript for the authors',
+                'accepted manuscripts published in eLife include the the decision letter from the editor to the authors, and the author responses to the decision letter',
             ], 'bullet'),
             new Paragraph('The overall aim is to make peer review faster, fairer and more open. <a href="'.$this->get('router')->generate('article', ['id' => '00855']).'">eLife does not support the Impact Factor</a> and is a co-founder of the <a href="https://sfdora.org/">Declaration on Research Assessment (DORA)</a>. Regularly updated metrics relating to the eLife editorial process are available in our <a href="https://reviewer.elifesciences.org/author-guide/journal-metrics">Author Guide</a>.'),
             ArticleSection::basic('Related links', 2,
                 $this->render(Listing::unordered([
-                    '<a href="'.$this->get('router')->generate('inside-elife-article', ['id' => 'e9091cea']).'">Peer Review: New initiatives to enhance the value of eLife’s process</a> (Michael Eisen, eLife Editor-in-Chief, November 2019)',
-                    '<a href="https://reviewer.elifesciences.org/author-guide/editorial-process">Author Guide</a>',
-                    '<a href="https://www.youtube.com/watch?v=quCG17jZW-w">eLife: Changing the review process</a> (video, February 2014)',
+                    '<a href="'.$this->get('router')->generate('inside-elife-article', ['id' => 'e9091cea']).'">New from eLife: Preprints and peer review</a>',
+                    '<a href="'.$this->get('router')->generate('inside-elife-article', ['id' => 'e9091cea']).'">New from eLife: What we have learned about preprints</a>',
+                    '<a href="'.$this->get('router')->generate('inside-elife-article', ['id' => 'e9091cea']).'">Peer Review: eLife implementing "Publish, then Review" model of publishing</a>',
```

Comment:
Just verifying that all three list items need to link to the same content?

https://elifesciences.org/inside-elife/e9091cea/peer-review-new-initiatives-to-enhance-the-value-of-elife-s-process

Reply:
Welled spotted, they're just placeholders for now. I'm waiting for Editorial to let me know when they've uploaded the 2 new items to the CMS and then I'll be updating those links to point at the the correct articles and then this will be ready to be merged.